.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Dispatch on Label
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node
+      - name: Install firefox
+        run: |
+          sudo apt install firefox
+          xvfb-run --auto-servernum
```

Comment:
Given that the action in the next step is supposed to install `xvfb`, this shouldn't work, right? And even if it does, what is it supposed to do? If it doesn't really do anything, i'd say lets remove it

Reply:
Should be removed.